src/en/foundations/colour/base.md

```diff
@@ -1,6 +1,9 @@
 ---
+title: Colour
 permalink: false
-tags: ['colourEN', 'header']
+tags: ["colourEN", "header"]
 ---
 
-# Colour
\ No newline at end of file
+# {{ title }}
```

Comment:
setting the title in front matter helps use the title as a piece of data. For example in the page title.